### PR TITLE
CC-34261 Split navigation for MP and BO (Fixathon).

### PR DESCRIPTION
**Pull request by ilyakubanov**
Ticket: https://spryker.atlassian.net/browse/CC-34311

### CODE REVIEW

**Review by dmiseev — DISMISSED** [review]

File: _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+        return [
+            new AclNavigationItemCollectionFilterPlugin(),
+        ];
+    }
+}
+```
+
+{% endinfo_block %}
+
+### 5) Separate login feature setup (security firewalls).
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md:333]: This title isn't clear, rephrase

File: _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+
+It requires upgrading `spryker/smyfony:3.5.0` and applying some changes on the project. For details, see [Symfony 5 integration](/docs/dg/dev/upgrade-and-migrate/upgrade-to-symfony-5.html).
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md:335]: Is this a call to action, prerequisites, or overview of this section?

File: _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+
+1. Install the required modules using Composer:
+
+```bash
+composer remove spryker/auth spryker/auth-mail-connector spryker/auth-mail-connector-extension spryker/authentication-merchant-portal-gui
+```
+
+```bash
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md:343]: Instructions missing

File: _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md [hunk 4]
```diff
@@ -0,0 +1,385 @@
...
+| MODULE       | DIRECTORY                    |
+|--------------|------------------------------|
+| Application  | vendor/spryker/application   |
+| MerchantUser | vendor/spryker/merchant-user |
+| Security     | vendor/spryker/security      |
+| Session      | vendor/spryker/session       |
+| User         | vendor/spryker/user          |
+| ZedUi        | vendor/spryker/zed-ui        |
+
+3. Apply changes from https://github.com/spryker-shop/suite/pull/681/files.
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md:358]: What do these changes do?

File: _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md (hunk 2 above)
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md:335]: prerequisites

File: _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md [hunk 5]
```diff
@@ -0,0 +1,384 @@
...
+console data:import merchant-user
+```
+
+### 3) Adjust the ACL
+
+By default, all newly created merchants and merchant users automatically have a group, role, and segment with autogenerated names based on `MerchantName`, `MerchantUserName`, and `MerchantPortalKey`.
+
+1. Adjust `AclConfig` to configure access to Merchant Portal's `*-merchant-portal-gui` pages:
+  * Give access to merchant users
+  * Optionally deny access for admin roles
```
[the remark this answers is not in the document]
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md:107]: According to business needs, Merchant portal can be for merchants only

File: _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md (hunk 3 above)
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md:343]: Instructions are higher for both of the console commands: `Install the required modules using Composer:`

File: _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md [hunk 6]
```diff
@@ -0,0 +1,386 @@
+
+
+{% info_block infoBox "" %}
+
+See [Install the Marketplace Merchant Portal Core feature](/docs/pbc/all/merchant-management/{{page.version}}/marketplace/install-and-upgrade/install-features/install-the-marketplace-merchant-portal-core-feature.html).
```
[the remark this answers is not in the document]
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md:5]: adjusted the doc

File: _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md [hunk 7]
```diff
@@ -0,0 +1,386 @@
...
+$ npm run mp:build
+```
+
+For production:
+
+```bash
+$ npm run mp:build:production
+```
+
+## Install backend
```
[the remark this answers is not in the document]
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md]: Renamed to `Install feature backend`, this is a standard section in IGs

File: _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md [hunk 8]
```diff
@@ -0,0 +1,382 @@
...
+**data/import/common/common/marketplace/merchant.csv**
+
+```csv
+merchant_key,merchant_reference,merchant_name,registration_number,status,email,is_active,url.de_DE,url.en_US
+sony-experts,MER000006,Sony Experts,HYY 134306,approved,michele@sony-experts.com,1,/de/merchant/sony-experts,/en/merchant/sony-experts
+```
+
+2. To integrate merchant user data import, follow [Install the Marketplace Merchant feature](/docs/pbc/all/merchant-management/{{page.version}}/marketplace/install-and-upgrade/install-features/install-the-marketplace-merchant-feature.html).
+
+**data/import/common/common/marketplace/merchant_user.csv**
```
[the remark this answers is not in the document]
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md:91]: adjusted the doc

File: _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md (hunk 1 above)
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md]: adjusted the doc

File: _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md (hunk 4 above)
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md]: adjusted the doc

File: _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md [hunk 9]
```diff
@@ -0,0 +1,391 @@
+
+
+{% info_block infoBox "" %}
+
+See [Install the Marketplace Merchant Portal Core feature](/docs/pbc/all/merchant-management/{{page.version}}/marketplace/install-and-upgrade/install-features/install-the-marketplace-merchant-portal-core-feature.html) to install the Marketplace Merchant Portal Core feature.
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md:5]: Why do I need to install it?

File: _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md [hunk 10]
```diff
@@ -355,7 +359,7 @@ composer require spryker/security-gui:"^1.0.0" spryker/security-merchant-portal-
 | User         | vendor/spryker/user          |
 | ZedUi        | vendor/spryker/zed-ui        |
 
-3. Apply changes from https://github.com/spryker-shop/suite/pull/681/files.
+3. Apply changes from https://github.com/spryker-shop/suite/pull/681/files to setup a separate login for Merchant Portal user.
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md:362]: This says the same as the title of the section. What changes does this PR contain?

**Pull request merged**